cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -190,7 +190,10 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
             inputDir, distinctPackageManagers, curationProvider, repositoryConfiguration
         ).mergeLabels(labels)
 
-        println("Found ${ortResult.getProjects().size} project(s) in total.")
+        val analyzedProjects = ortResult.getProjects()
+        val countPerType = analyzedProjects.groupingBy { it.id.type }.eachCount()
+        val projectCountDetails = countPerType.toSortedMap().map { (type, count) -> "$type ($count)" }.joinToString()
+        println("Found ${analyzedProjects.size} project(s): $projectCountDetails")
```

Comment:
As `projectCountDetails` might become a bit long in case of many types, how about using `\n` as separator?
(Also for readability)

Reply:
Hmm, I believe cases where more than, say, 2-3 package managers are used in a single repo are rather rare, or? And these should still fit into a single line.